xbmc/pvr/PVRManager.h

```diff
@@ -70,6 +71,21 @@ namespace PVR
     ManagerStateStarted
   };
 
+  struct PVRManagerEvent
+  {
+    virtual ~PVRManagerEvent() {};
+  };
+
+  namespace PVRManagerEvents
+  {
+    struct Starting    : PVRManagerEvent {};
```

Comment:
but yeah, i guess i will simplify this. if we need to carry data along with the events, we easily can change that later.